### ISSUE TITLE
Quick fix for covariance and contravariance

### ISSUE THREAD

**Issue by HerrNiklasRaab** [dart-lang/sdk#41540]
Once this lands: https://medium.com/dartlang/dart-declaration-site-variance-5c0e9c5f18a5 it would be nice to have quick fixes for below scenarios:

``` dart
// Before quick fix
abstract class Test<T> {
  void test(T a);
}

// After quick fix
abstract class Test<in T> {
  void test(T a);
}
```
``` dart
// Before quick fix
abstract class Test2<T> {
  T test();
}

// After quick fix
abstract class Test2<out T> {
  T test();
}
```

**Comment by jcollins-g** [comment]
Given that [the variance implementation](https://github.com/dart-lang/sdk/issues/48906) is, as of time of writing, not fully specified, nor fully implemented elsewhere, nor a part of a Dart release schedule or active release, it seems reasonable to remove the diagnostic-without-fix label as no user outside of those trying the experimental implementation will see these diagnostics.  This can be unblocked when/if an implementation issue for the analyzer for the variance feature is created and complete.
